src/model/Global.ts

```diff
@@ -431,6 +433,18 @@ echarts.use([${seriesImportName}]);`);
                         return;
                     }
 
+                    // Before multiple tooltips get supported,
+                    // we do this check to avoid unexpected exception.
+                    if (mainType === 'tooltip') {
+                        if (tooltipExists) {
+                            if (__DEV__) {
+                                warn('Currently only one tooltip component is allowed.');
```

Comment:
Here we should tweak slightly, or it will output the log as many times as **the count of tooltip option - 1**.

Reply:
@pissang Done. Now it only outputs the warning once when `setOption` is called, no matter how many tooltips are provided.